.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @open-turo/node
```

Comment:
Interestingly, github complains about the group 🤔

Reply:
I can see `open-turo/github-actions` but not `open-turo/node`. Maybe we should update it

Reply:
Changing to open-turo/github-actions for now. This is because that team hasn't been addded to this repo